src/yunohost/tools.py

```diff
@@ -42,8 +42,16 @@
 from moulinette import msettings, msignals, m18n
 from moulinette.core import MoulinetteError, init_authenticator
 from moulinette.utils.log import getActionLogger
+<<<<<<< b60d8ca822d08c8e3fdf8a17505ff3e285b28164
 from moulinette.utils.process import check_output
 from moulinette.utils.filesystem import read_json, write_to_json
+=======
+from moulinette.utils.filesystem import read_json, write_to_json, read_file
+<<<<<<< a934b3fd19403859cf6a46713b82945524f9ac28
+>>>>>>> [mod] move spectre-meltdown check to diagnosis function
+=======
+from moulinette.utils.process import check_output
+>>>>>>> [enh] uses speed47 script to check for meltdown vulnerability
```

Comment:
Looks like there was a conflict resolution "oops"... ;-)

Reply:
weird, I though I had fixed that :x (I had a bug when I've tested that, then commit and pushed but apparently something went wrong >.<)

Reply:
okay, find it, I've pushed ... the admin, not the core :sweat_smile: 

Thanks @JimboJoe and @M5oul :heart: